### PR TITLE
Create interest rate comparison view

### PR DESCRIPTION
**Pull request by zane-programs**
I have now completed an MVP version of the interest rate comparison view that I proposed [on Snapshot](https://vote.rari.capital/#/rari/proposal/QmeQzoUJkSsATmhLFf5dkfUq2q5dCYANkwrAraMkezzkeJ). If you'd like, you can take a look at my work using [this live link](https://rari-zane-work.vercel.app/interest_rates).

If you notice any important issues, please let me know—I made a major refactor during a Monday-Tuesday all-nighter in order to improve performance.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "noEmit": true,
     "jsx": "react-jsx",
     "noFallthroughCasesInSwitch": true,
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "downlevelIteration": true
```
Comment (sharad-s) [inline tsconfig.json:23]: Can you explain what this change in `tsconfig.json` is for?

Comment (zane-programs) [reply tsconfig.json:23]: I added `downlevelIteration` in order to support the use of the spread operator on a `Set`. In `InterestRatesView.tsx`, I used it like this: `const tokenAddresses = [...new Set(allTokens)];`

You can see more about `downlevelIteration` in the [tsconfig reference](https://www.typescriptlang.org/tsconfig#downlevelIteration).

Comment (sharad-s) [reply tsconfig.json:23]: Awesome was trying to implement this exact thing yesterday. thanks

**Comment by zane-programs** [comment]
Just fixed the version on the `window-table` package, which should fix the error given by Cypress.

**Pull request merged**